docs/Developers/Newcomers_Guide.rst

```diff
@@ -233,33 +238,39 @@ Step 6. Creating a ``Pull Request``
 -----------------------------------
 
 Now that your commit has been sent to your fork, it is time
-to do a ``Pull Request``. It can be done by accessing your fork on GitHub and
-clicking ``New Pull Request``.
+to do a ``Pull Request``.
+It can be done by accessing your fork on GitHub and clicking
+``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
 .. note::
 
     Do not delete your comments on Github because it makes it hard for other
-    developers to follow on that issue. If necessary, edit your comment in case
-    there is a typo or a task list to be updated. If you have to add some new
-    information, make a new comment.
+    developers to follow on that issue.
+    If necessary, edit your comment in case there is a typo or a task list to
+    be updated.
+    If you have to add some new information, make a new comment.
 
 If you know you have more work to do on this ``Pull Request`` before it is
-ready to be accepted, you can optionally indicate this to other
-developers by starting your ``Pull Request`` title with ``wip``
-(case-insensitive).
+ready to be accepted, you can optionally indicate this to other developers
+by starting your ``Pull Request`` title with ``wip`` (case-insensitive).
 
 Step 7. Waiting for Review
 --------------------------
 
-After creating a Pull Request, your PR is open to the review process (to read
-more about it, have patience and it is explained on the next step), and all
-you can do is wait. The best thing you can do while at this step is review
-other people's PRs. Not only will this help the maintainers with the workload,
-but this is one of the three core steps towards becoming a full-norm coalaian.
+After creating a Pull Request, your PR is open to the review process.
+
+..note::
+
+    Do not ask for reviews in the chat. Instead go to http://coala.io/review and
+    do a review yourself.
```

Comment:
maybe add something like "you may ask for a review after you have done a review as well. When you ask for it, please provide a link of something you just reviewed. We need to do this so everyone gets timely reviews and people who are making this happen get preferred"